File: app/src/main/java/com/winsun/fruitmix/file/data/station/FileDataSource.java
package com.winsun.fruitmix.file.data.station;

import com.winsun.fruitmix.callback.BaseLoadDataCallback;
import com.winsun.fruitmix.file.data.model.AbstractRemoteFile;

/**
 * Created by Administrator on 2017/7/27.
 */

public interface FileDataSource {

    void getFile(final String folderUUID, String rootUUID, final BaseLoadDataCallback<AbstractRemoteFile> callback);

}

File: app/src/main/java/com/winsun/fruitmix/newdesign201804/file/data/FileDataSource.kt
package com.winsun.fruitmix.newdesign201804.file.data

import com.winsun.fruitmix.callback.BaseLoadDataCallback
import com.winsun.fruitmix.file.data.model.AbstractRemoteFile

public interface FileDataSource{

    fun getFile(baseLoadDataCallback: BaseLoadDataCallback<AbstractRemoteFile>)

}